files/zh-cn/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md

```diff
@@ -3,19 +3,27 @@ title: Web Workers 可以使用的函数和类
 slug: Web/API/Web_Workers_API/Functions_and_classes_available_to_workers
 ---
 
-{{DefaultAPISidebar("Web Workers API") }}
+{{DefaultAPISidebar("Web Workers API")}}
 
 除了标准的 [JavaScript](/zh-CN/docs/Web/JavaScript) 函数集（例如 {{jsxref("String")}}、{{jsxref("Array")}}、{{jsxref("Object")}}、{{jsxref("JSON")}} 等）DOM 中的许多函数都可在 worker 中使用。本文列出了这些函数。
 
 ## Worker 上下文和函数
 
 **Worker 在另一个全局上下文中运行，与当前的 window 不同！**{{domxref("Window")}} 并不直接在 worker 中可用，其中的很多方法都通过共享的混入（`WindowOrWorkerGlobalScope`）定义，并通过 worker 派生的 {{domxref("WorkerGlobalScope")}} 上下文提供这些方法：
 
-- {{domxref("DedicatedWorkerGlobalScope")}}——专用 worker
-- {{domxref("SharedWorkerGlobalScope")}}——共享 worker
-- {{domxref("ServiceWorkerGlobalScope")}}——[service worker](/zh-CN/docs/Web/API/Service_Worker_API)
+- {{domxref("DedicatedWorkerGlobalScope")}} 用于专用 worker
+- {{domxref("SharedWorkerGlobalScope")}} 用于共享 worker
+- {{domxref("ServiceWorkerGlobalScope")}} 用于 [service workers](/zh-CN/docs/Web/API/Service_Worker_API)
```

Comment:
https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%A4%8D%E6%95%B0%E5%BD%A2%E5%BC%8F

same problem in another PR

```suggestion
- {{domxref("ServiceWorkerGlobalScope")}} 用于 [service worker](/zh-CN/docs/Web/API/Service_Worker_API)
```

Reply:
Have been modified